starwars/src/components/StarCardDisplay.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import StarCard from "./StarCard";
+import { Container, Row } from "reactstrap";
+
+export default function StarsData() {
+  const [heros, setHeros] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://swapi.co/api/people/")
+      .then(response => {
+        console.log(response.data.results);
+        setHeros(response.data.results);
+      })
+      .catch(error => {
+        console.log("the data was not returned", error);
+      });
+  }, []);
+
+  return (
+    <Container>
+      <Row>
+        {heros.map(item => {
+          //  console.log(item.name)
+
+          console.log(item.mass);
+          return (
+            <div>
+              <div class="row">
+                <div class="col">
+                  <StarCard
```

Comment:
keep in mind that you can also pass the whole object as a prop (`item={item}`) and access the keys the same way you normally would